app/brave_generated_resources.grd

```diff
@@ -277,28 +277,28 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
         Shields
       </message>
       <message name="IDS_SETTINGS_BRAVE_SHIELDS_DEFAULTS_TITLE" desc="The title for Brave shields Defaults sub-section in settings">
-        Defaults
+        Global Defaults
       </message>
       <message name="IDS_SETTINGS_BRAVE_SHIELDS_DEFAULTS_DESCRIPTION" desc="The description for Brave shields Defaults sub-section in settings">
-        Your current per-site settings will be retained.
+        These are the default Shields settings for new sites. Changing these won't affect your existing per-site settings.
```

Comment:
@tomlowenthal should we be using the word 'website' instead of 'site'? Also is it clear enough to say 'new sites'? It doesn't infer _visiting_ that site explicitly and clearly. If you want suggestions, then how about something like _"These are the default Shields settings when visiting websites. Changing these won't...."_?

Reply:
How about:
> "These are the default Shields settings. They apply to all websites unless you change something in the Shields panel on a particular site. Changing these won't affect your existing per-site settings."